src/utils/format.ts

```diff
@@ -7,4 +7,13 @@ export const capitalizeChar = (str: string) => {
     .join(' ');
 };
 
-export const test = {};
+export const formatDate = (str: string) => {
+  const newDate = new Date(str);
+  const date = `${newDate.getDate()}/${newDate.getMonth()}/${newDate.getFullYear()}`;
+  const hour = newDate.getUTCHours();
+  const minute = `:${newDate.getUTCMinutes()}`;
+  const newTime = `${hour > 12 ? hour - 12 : hour < 1 ? 12 : hour}${minute}${
+    hour > 11 ? 'pm' : 'am'
+  }`;
+  return { getDate: date, getTime: newTime };
+};
```

Comment:
change the const date to const getDate
change the newTime to const geTime
Then in line 18 return {getDate, geTime }

Reply:
> change the const date to const getDate change the newTime to const geTime Then in line 18 return {getDate, geTime }

alright noted.